### ISSUE TITLE
Notification showing server IP/Port

### ISSUE THREAD

**Issue by HeyItsJono** [wolpi/prim-ftpd#139]
Hey there,
I love the ability to start a server from a quicktile, but since I change wifi networks a lot, I still end up having to open the app each time to check what the server IP is so I can type it into my client. Would you consider adding an option to display a notification with the server IP/port while the server is switched on?

Or failing that, maybe change the text below the quicktile from 'pFTPD Runnning ..." to the IP/Port combo when the server is turned on? Or perhaps even a quicktile long-press action which would show a popup with the IP/Port.

Basically anything to avoid having to open the app each time would be absolutely wonderful! Thanks!

**Comment by wolpi** [comment]
Sounds interesting, I'm going to look into it :+1:

**Comment by HeyItsJono** [comment]
Thanks so much!
